File: GrowthWare_CS/GrowthWareFramework/ModelObjects/MWorkFlowProfile.cs

using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Diagnostics;
using System.Web;
using GrowthWare.Framework.ModelObjects.Base.Interfaces;
using GrowthWare.Framework.ModelObjects.Base;

namespace GrowthWare.Framework.ModelObjects
{
	[Serializable(), CLSCompliant(true)]
	public class MWorkFlowProfile : MProfile
	{

		/// <summary>
		/// Will return a workflow profile with the default vaules
		/// </summary>
		/// <remarks></remarks>

		public MWorkFlowProfile()
		{
		}

		public MWorkFlowProfile(DataRow dr)
		{
			base.init(dr);
		}
	}
}

File: GrowthWare_VB/GrowthWareFrameworkModel/Profiles/MWorkFlowProfile.vb
Imports GrowthWare.Framework.ModelObjects.Base.Interfaces
Imports GrowthWare.Framework.ModelObjects.Base

Namespace ModelObjects
	<Serializable(), CLSCompliant(True)> _
	Public Class MWorkFlowProfile
		Inherits MProfile
		Implements IProfile

		''' <summary>
		''' Will return a workflow profile with the default vaules
		''' </summary>
		''' <remarks></remarks>
		Public Sub New()

		End Sub

		Public Sub New(ByVal dr As DataRow)
			MyBase.Init(dr)
		End Sub
	End Class
End Namespace